test/quantization/test_backward_compatibility.py

```diff
@@ -18,7 +18,7 @@
 # JIT tests use double as the default dtype
 torch.set_default_dtype(torch.double)
 
-class TestSerialization(TestCase):
+class TestBC(TestCase):
```

Comment:
what does BC stand for?

Reply:
backward compatibility, this is too long so I abbreviated it to BC

Reply:
turns out I need to change this back because the test name is in the serialized file